test/framework/src/main/java/org/elasticsearch/entitlement/initialization/TestEntitlementInitialization.java

```diff
@@ -38,17 +37,19 @@
 public class TestEntitlementInitialization {
 
     private static ElasticsearchEntitlementChecker checker;
+    public static InitializeArgs initializeArgs;
 
     // Note: referenced by bridge reflectively
     public static EntitlementChecker checker() {
         return checker;
     }
 
     public static void initialize(Instrumentation inst) throws Exception {
-        TestEntitlementBootstrap.BootstrapArgs bootstrapArgs = TestEntitlementBootstrap.bootstrapArgs();
-        checker = EntitlementInitialization.initChecker(inst, createPolicyManager(bootstrapArgs.pathLookup()));
+        checker = EntitlementInitialization.initChecker(inst, createPolicyManager(initializeArgs.pathLookup()));
     }
 
+    public record InitializeArgs(PathLookup pathLookup) {}
```

Comment:
`TestInitializationArgs`?

Reply:
Ditto. The problem this solves is that we can't pass arguments directly to the `initialize` method.